samples/dnn/custom_layer.py

```diff
@@ -0,0 +1,31 @@
+import cv2 as cv
```

Comment:
The new approach destroys original sample with custom layer. Formally, all code blocks are there, but the sample does not create Net and does not use the new layer.